backward.hpp

```diff
@@ -24,6 +24,8 @@
 #ifndef H_6B9572DA_A64B_49E6_B234_051480991C89
 #define H_6B9572DA_A64B_49E6_B234_051480991C89
 
+#include "backward_config.hpp"
```

Comment:
its manual *if and only if* you **need** to check for new updates. if you never care about new updates or if you want to easily hack your own version, that's the way to go.

If adding a guard macro keeps the simple "copy/paste" style working while supporting your use case, then please do and I will be happy to merge this PR.

(and sorry for the delay in responding)